src/main/service/createAccountService.ts

```diff
@@ -1,6 +1,33 @@
 import { allowedImageTypes } from '../models/consts';
 
 export class CreateAccountService {
+
+  public async uploadCreateAccount(data: any): Promise<any> {
+    this.generatePublicationUploadHeaders(data);
+    return 'ABCD1234';
+    //TODO: uncomment after PUB-1079
+    // (data: any)
+    // return await accountManagementRequests.uploadNewAccountRequest(
+    //   data,
+    //   this.generatePublicationUploadHeaders(data),
+    // );
+  }
+
+  public generatePublicationUploadHeaders(headers): object {
+    return {
+      'x-provenance': 'CREATE_ACCOUNT',
+      'x-source-file-name': headers.fileName,
+      'x-full-name': headers['fullName'],
+      'x-email-address': headers['emailAddress'],
+      'x-employer': headers['employer'],
+      'Content-Type': 'multipart/form-data',
+    };
+  }
+  public getFileExtension(fileName: string): string {
```

Comment:
can you not use functions from the manual upload? no point of duplicating logic which is exactly the same